script/configs/mainnet/deploy_mainnet_config.json

```
@@ -0,0 +1,3 @@
+{
+    "risc_zero_verifier_address": "0x8EaB2D97Dfce405A1692a21b3ff3A172d593D319"
```

It seems odd to me to use `snake_case` in a JSON file. I usually see `camelCase` or `kebab-case`. I'd lean towards using `camelCase` here because Solidity uses it.
```suggestion
    "riscZeroVerifierAddress": "0x8EaB2D97Dfce405A1692a21b3ff3A172d593D319"
```